### PR TITLE
Meta: Make bundles smaller

### PR DESCRIPTION
**Pull request by fregante**
Fixes https://github.com/sindresorhus/refined-github/issues/2996

I followed every step explained in that issue. This is the result:

```vb
           background.js ⏤  10.9 kB (-773 B)
 browser-polyfill.min.js ⏤  3.07 kB
             options.css ⏤  1.3 kB
            options.html ⏤  1.02 kB
              options.js ⏤  33.3 kB (-3.33 kB)
      refined-github.css ⏤  10.8 kB
       refined-github.js ⏤  96.1 kB (-5.12 kB)
    resolve-conflicts.js ⏤  1.6 kB (-34 B)
```

Once https://github.com/webpack/webpack/issues/10720 is resolved, we'll have further savings (and performance improvements):

```vb
           background.js ⏤  10.5 kB (-361 B)
 browser-polyfill.min.js ⏤  3.07 kB
             options.css ⏤  1.3 kB
            options.html ⏤  1.02 kB
              options.js ⏤  28.6 kB (-4.66 kB)
      refined-github.css ⏤  10.8 kB
       refined-github.js ⏤  94.6 kB (-1.54 kB)
    resolve-conflicts.js ⏤  1.59 kB (-16 B)
```

Rollup makes even smaller bundles, but alas #3019 
Native ES modules support [in Content Scripts](https://medium.com/front-end-weekly/es6-modules-in-chrome-extensions-an-introduction-313b3fce955b#d45a) would be amazing (together with Snowpack) but that seems far away: https://bugzilla.mozilla.org/show_bug.cgi?id=1451545

### CODE REVIEW

**Comment by yakov116** [comment]
`refined-github.css ⏤  10.9 kB (+88 B)`

Curious why

**Comment by fregante** [comment]
Heh, probably because this PR is based on a different commit, so those 88 bytes are just the diff between the 2 CSS files.

**Comment by fregante** [comment]
> so those 88 bytes are just the diff between the 2 CSS files.

Confirmed. First post updated

File: webpack.config.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /// <reference types="./source/globals" />
```
Comment (fregante) [inline webpack.config.ts:1]: `webpack.config.ts` is the only file with non-search-and-replace changes

**Pull request merged**